src/uucore/src/lib/features/checksum.rs

```diff
@@ -550,15 +627,14 @@ fn get_input_file(filename: &OsStr) -> UResult<Box<dyn Read>> {
 
 /// Extracts the algorithm name and length from the regex captures if the algo-based format is matched.
```

Comment:
The comment is outdated because the regex captures have been replaced by `LineInfo`.
```suggestion
/// Gets the algorithm name and length from the `LineInfo` if the algo-based format is matched.
```